instrumentation/src/com/intellij/rt/coverage/instrumentation/kotlin/KotlinUtils.java

```diff
@@ -28,4 +40,29 @@ public static boolean isKotlinClass(Instrumenter context) {
     context.addProperty(KOTLIN_CLASS_LABEL, isKotlin);
     return isKotlin;
   }
+
+  private static final boolean ourKotlinEnabled = !"false".equals(System.getProperty("coverage.kotlin.enable", "true"));
```

Comment:
maybe better use java.lang.Boolean#getBoolean or at least java.lang.Boolean#parseBoolean

Reply:
With `java.lang.Boolean#getBoolean` I cannot have `true` default value.
In case of
```
private static final boolean ourKotlinEnabled = Boolean.parseBoolean(System.getProperty("coverage.kotlin.enable", "true"));
```
Is it ok that `-Dcoverage.kotlin.enable=tru` will disable Kotlin?
Should I care about incorrect strings passed as property value?

Reply:
This is really minor, if it produce problems - leave it as is